index.Rmd

```diff
@@ -103,15 +103,27 @@ You will be asked to give your credentials (username + password) and then the (a
+The first time a Juno pipeline is executed, the preparation might take longer than expected. This is due to the installation of the basic software that is required for the pipeline to run. Be patient! You can recognize that the pipeline is still preparing when it shows a blinking box ([Figure 1.5](#installation-fig) step 3) or when it keeps printing lines inside the terminal.
 
-Once your pipeline is finished you should see some final messages informing you that 100% of the steps were finished and that your output was created. 
+```{r installation-fig, echo = FALSE, eval = TRUE, fig.cap = "Screenshot of the Juno-typing pipeline in the terminal. The pipeline asks to create an environment(step 1). User input is required. To proceed the pipeline press y followed by enter. If executed correctly the pipeline will now show that it is creating the environment(step 2). The blinking box at the bottom indicates that the pipeline is still installing/updating(step 3)."}
+knitr::include_graphics("figures/show_installation.png")
+```
+
+After installation the Juno pipeline will start running. Now the terminal shows <span style="color:#f4d03f;">yellow</span> and/or <span style="color:#00b16a;">green</span> messages. These messages indicate that the pipeline is being executed ([Figure 1.6](#snakemake-complete-run-fig)). At the start of the yellow messages the number of steps is shown ([Figure 1.6](#snakemake-complete-run-fig) step 1).  The amount of steps indicate how much time it will take for the pipeline to finish. Once the pipeline is finished there will be a message printed that all the steps are performed ([Figure 1.6](#snakemake-complete-run-fig) step 2).
+
+```{r snakemake-complete-run-fig, echo = FALSE, eval = TRUE, fig.cap = "Screenshot of the Juno AMR pipeline in the terminal. The pipeline is executing all required jobs to create the requested results. The terminal shows a list of all the jobs that will be executed(step 1). Furthermore there is green and yellow text that shows the steps of the pipeline. At the bottom of the terminal there is a line that shows the pipeline is finished(step 2)."}
+knitr::include_graphics("figures/snakemake_complete_run.png")
+```
+
+If a step fails, you will likely see <span style="color:#d64541;">red</span> text appearing on the screen ([Figure 1.7](#snakemake-complete-run-fig)). The pipeline might proceed the other steps. If the pipeline fails check the section [General Troubleshooting](#general-troubleshooting) for more help or any other problems you may encounter.
 
-See the section [General Troubleshooting](#general-troubleshooting) for any problems you may encounter. 
+```{r snakemake-error-fig, echo = FALSE, eval = TRUE, fig.cap = "Screenshot of the Juno AMR pipeline in the terminal. The pipeline failed on one of the steps. The red text shows an error message(step 1)."}
+knitr::include_graphics("figures/screenshot_snakemake_error.png")
```

Comment:
Same here... the output is barely legible. Maybe only include the bottom half of the screenshot? And is it possible to 'zoom in'  or make the font larger in the terminal? I think the screenshot is taken with very bad resolution (and probably even lowered when putting it inside the html. Maybe having larger font helps! Not sure how to do that though 😅